### DISCUSSION TITLE
Immersive skin on windows 8.1

### DISCUSSION THREAD

**Discussion by Voyager-III** [Open-Shell/Open-Shell-Menu#1866] [General]
When Open-Shell-Menu is installed on windows 8.1 the Immersive skin is missing? But not on Windows 11, I copied over the skin from my windows 11 installation and it seem's to work great on 8.1, I was wondering if this is left out for a reason or is it a bug in the installation when installed on Windows 8.1?

**Comment by ge0rdi** [comment]
Could it be that you have some older `Open-Shell` version on Win 8.1 machine?
Immersive skin was added in `4.4.191`.
It is by default selected on Win 10/11, but you should be able to select it also on older Windows.

**Reply by Voyager-III** [reply]
Hi, No I am using the lastest 4.4.191 installer on Windows 8.1 but for some reason the installer does not install the immersive skin file into the skins directory all the other skin files are there but not immersive, I have to copy it over from the skin's directory on my Windows 11 computer which is also running 4.4.191, I have tried installing it on several windows 8.1 computers but with the same result, It does not make any sense?

**Reply by ge0rdi** [reply]
Oki, I see now.
We have condition in installer to install the skin only on Win10 and higher.

Not sure why it is like that :(
I guess there should be no problem to install it on all systems (it will be still selected by default on Win10+).

**Reply by ge0rdi** [reply]
I have updated installer script.
Since next version the skin should be installed on all systems.

**Reply by EJSnow** [reply]
You forgot that Windows 7 doesn't support Metro colors so the Immersive skin won't work there. Really, the condition should be to install the skin only on Windows 8 or higher. Open-Shell still supports Windows 7, in case you forgot.

**Reply by ge0rdi** [reply]
Thank you for noticing.
I can change the condition, no problem.
Though I guess there is no big deal even if the skin is installed. Nobody will simply use it on Win7.

BTW Have you tried to use the skin on Win7? Doesn't it work at all?

**Reply by EJSnow** [reply]
I haven't but I can try. I'll let you know what I find.

**Reply by EJSnow** [reply]
(nevermind see below)

**Comment by bonzibudd** [comment]
@ge0rdi 

Like Win761 said, it won't work on Windows 7 at all due to the lack of metro colors. 

Ivo coded Classic Shell to immediately revert to the default skin if a Windows 7 system tries to run anything with metro colors.

However, I did code the colors on the immersive skin to be compatible with Windows 8, so it can still be supported.

**Reply by ge0rdi** [reply]
Cool, thank you for explanation.
I have updated installer script (https://github.com/Open-Shell/Open-Shell-Menu/commit/211d12be706894d0515f6bc16f7f5f1693a836aa).
